src/Adapter/Presenter/Object/ObjectPresenter.php

```diff
@@ -52,6 +52,11 @@ public function present($object)
         foreach ($fields as $fieldName => $null) {
             $presentedObject[$fieldName] = $object->{$fieldName};
         }
+
+        if (get_class($object) === 'Product') {
```

Comment:
If merchant uses an override that extends `Product` class, this might not work ? 🤔 

Could we use something that works with children classes, like `instanceof` ?

Reply:
Bien vu ! Thanks @matks